contribute/style-guides/documentation-style-guide.md

```diff
@@ -192,6 +192,14 @@ Authorization: Bearer eyJrIjoiT0tTcG1pUlY2RnVKZTFVaDFsNFZXdE9ZWmNrMkZYbk
 
 Grafana products has some words, abbreviations, and terms particular to the Grafana discourse community.
 
+#### changelog
+
+One word, not two.
+
```

Comment:
If referring to the official changelog file in the context of releases, the official name is CHANGELOG.md.

Reply:
Are you suggesting that we add it as an exception? 

```
**Exceptions:**

- When referring to the file containing the official changelog, use the filename: CHANGELOG.md
```

Reply:
Yes, exactly. I like your format better.

Reply:
`CHANGELOG.md`

Reply:
Yes, perfect